LocalSettings.php

```diff
@@ -3956,6 +3956,36 @@
 			'css' => [],
 		],
 	],
+	
+	// TextExtracts
+	'wgExtractsRemoveClasses' => [
+		'default' => [
+			'table',
+			'div',
+			'script',
+			'input',
+			'style',
+			'ul.gallery',
+			'.mw-editsection',
+			'sup.reference',
+			'ol.references',
+			'.error',
+			'.nomobile',
+			'.noprint',
+			'.noexcerpt',
+			'.sortkey',
+		],
+		'gratispaideiawiki' => [
```

Comment:
This should inherit the defaults also probably (+gratispaideiawiki)

Reply:
Do you mean, all the defaults should also be included under +gratispaideiawiki?

Reply:
No, I just meant you needed to add the `+` which inherits the default, and appends to it with the new array.

Reply:
@Universal-Omega just as i've done?

Reply:
Yes